api/v1alpha1/inferencepool_types.go

```diff
@@ -61,8 +61,100 @@ type InferencePoolSpec struct {
 	// +kubebuilder:validation:Maximum=65535
 	// +kubebuilder:validation:Required
 	TargetPortNumber int32 `json:"targetPortNumber"`
+
+	// EndpointPickerConfig selects and configures the endpoint picking algorithm to apply on the requests sent
```

Comment:
How does the term "picker" differ from "selector"? I ask b/c users have established a mental model for Endpoint Selector. Unless "picker" has a different semantic meaning than "selector", I'd like to see the project standardize on "selector".